### ISSUE TITLE
time { ruby doit.rb; }

### ISSUE THREAD

**Issue by stoivo** [idank/explainshell#168]
Command  `time { ruby doit.rb; }`
Should not blow up

<img width="727" alt="screen shot 2016-12-13 at 10 03 57" src="https://cloud.githubusercontent.com/assets/3492040/21133922/84c87ca8-c11b-11e6-93eb-ebc07eded2c2.png">

**Comment by idank** [comment]
Indeed. There are other bash builtins that aren't supported either.

If you want to work on this, start here: https://github.com/idank/bashlex/blob/master/bashlex/parser.py#L520
